linter_api/src/ast/ty/enum_ty.rs

```diff
@@ -60,21 +57,21 @@ impl<'ast> EnumTy<'ast> {
 #[derive(Debug, PartialEq, Eq, Hash)]
 pub struct EnumItem<'ast> {
     cx: &'ast AstContext<'ast>,
-    def_id: DefTyId,
+    def_id: TyDefId,
     name: SymbolId,
     kind: VariantKind<'ast>,
     // FIXME: Add optional expression for variant number
 }
 
 #[cfg(feature = "driver-api")]
 impl<'ast> EnumItem<'ast> {
```

Comment:
`Item` endings should be reserved to items, maybe end with `Ty` or something else? Or we could remove this, see comment above.